ifopt/CMakeLists.txt

```diff
@@ -1,4 +1,63 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(ifopt)
-find_package(catkin REQUIRED)
-catkin_metapackage()
+option(CATKIN_BUILD "CATKIN_BUILD" ON)
+MESSAGE(STATUS "CATKIN_BUILD is " ${CATKIN_BUILD})
+
+if(${CATKIN_BUILD})
+    find_package(catkin REQUIRED)
```

Comment:
This actually returns a variable CATKIN_FOUND that could be used instead of the CATKIN_BUILD, that way this flag does not have to be set by the user.